helpers/GSGetter.ts

```diff
@@ -201,6 +203,68 @@ export class GCGetter {
                 const id = await persistence.connect_user_to_calendar_id(calendar[1], context.getSender());
                 const final_calendar_id = await persistence.get_preferred_calendar_id(context.getSender());
                 break;
+
+            case (Command.public):
+
+                const user_id = await read.getRoomReader().getMembers(context.getRoom().id);
+                let email_ids: Array<any> = user_id;
```

Comment:
This line is assigning a variable to another without changing anything, is that the intended behavior?

Reply:
Changed to 
`let email_ids : Array<any> = []`

Reply:
Hm. But you could've used the `user_id` variable since it has _the exact same content_

Reply:
`user_id` contains not just  the email ids of all the users  but also the username and other variables. So in an further down the code, I tried extracting only the email ids in the array `email_ids` since I needed a separate array which just consists of email ids to pass in the api request.

Reply:
Well then, there is no point in assigning the `user_id` value to the `email_ids` variable - you could've just defined it as `const email_ids: Array<any>`

Reply:
Yes actually it was just to avoid the above mentioned error, but now I have eliminated and initialized it as an empty array. 
`let email_ids : Array<any> = []`